includes/components/class-review.php

```diff
@@ -251,9 +254,35 @@ public function alter_listing_manage_menu( $items, $menu ) {
 			$listing = $menu->get_context( 'listing' );
 
 			if ( $listing && $listing->get_rating_count() ) {
+
+				// Set query arguments.
+				$query_args = [
+					'type'           => hp\prefix( 'review' ),
+					'status'         => 'approve',
+					'posts_per_page' => -1,
+					'count'          => true,
+					'post_id'        => $listing->get_id(),
+				];
+
+				// Get cache group.
+				$cache_group = hivepress()->model->get_cache_group( 'comment', hp\prefix( 'review' ) );
+
+				// Get cached review count.
+				$review_count = hivepress()->cache->get_cache( array_merge( $query_args, [ 'format' => 'reviews' ] ), $cache_group );
```

Comment:
yes, i left a comment below (check it first), it seems that you use global cache but it's better to use per-listing cache (helpers with "..._post_cache"), then the changed review count for one of the listings will not reset others.